server.js

```diff
@@ -15,7 +15,7 @@ const app = express();
 app.use(bodyParser.json());
 
 const port = process.env.PORT || 5001;
-const configPath = process.env.SERVER_EXTRA_CONFIG_FILE || '/etc/webapp/customServerConfig.json';
+const configPath = process.env.SERVER_EXTRA_CONFIG_FILE || './etc/webapp/customServerConfig.json';
```

Comment:
This would always point to a relative path inside the repo, I don't think we want this. The current main use of this env var places it's config in the path `/etc/webapp/customServerConfig.json` so I think we're good to keep this the same.

If a user wants to use the relative path instead, they can still use the `SERVER_EXTRA_CONFIG_FILE ` env

Reply:
OK,  that makes sense, but at least we know the code works now.